src/App.js

```diff
@@ -1,9 +1,106 @@
+import { useState } from "react";
+import './style.css'; //이번 과제는 컴포넌트 분리가 어려워서 css를 사용했습니다 ㅠㅠ
+import './background.js';
+import Clock from './clock.js';
 
 function App() {
+  const [toDo, inputToDo] = useState("");
+  const [toDos, setToDos] = useState([]);
+  const [doneToDos, movetoDos] = useState([]); 
+
+  //inputToDo를 통해 입력받음
+  const onChange = (event) => inputToDo(event.target.value);
+
+
+  const submitInput = (event) => {
+      event.preventDefault();
+      if (toDo === "") {
+        return;
+      }
+      setToDos((currentArray) => [toDo, ...currentArray]); //입력받은 걸 배열에 넣음
+      inputToDo("");
+
+    };
+
+    const moveDoneButton = (item,index) =>{
+    
+      movetoDos((current) => [item, ...current]); //Done 리스트의 배열로 추가됨
+      setToDos(toDos.filter((item, todoIndex) => index !== todoIndex)); //Todo에서는 삭제
+      
+  
+    }
+
+    const deleteToDoButton = index => {
+      setToDos(toDos.filter((item, todoIndex) => index !== todoIndex)); //Todo 삭제
+      
+    };
+
+    const moveToDoButton = (item,index) =>{
+   
+      setToDos((current) => [item, ...current]); //TODOlist 배열로 추가됨
+      movetoDos(doneToDos.filter((item, todoIndex) => index !== todoIndex));
+      
+  
+    }
+
+    const deleteDoneButton = index => {
+      movetoDos(doneToDos.filter((item, todoIndex) => index !== todoIndex)); //Done리스트 삭제
+      };
+  
+
   return (
-    <div>
-      <h1>15기 화이팅~</h1>
-    </div>
+    <div className="center">
+    <Clock></Clock>
```

Comment:
이렇게 태그 사이에 어떠한 내용도 존재하지 않는 경우  React에서는 `self-closing` 형식으로 사용할 수 있답니다

```suggestion
    <Clock />
```

Reply:
감사합니다!! 코드에 그대로 반영하겠습니다^-^